pkg/framework/image.go

```diff
@@ -42,6 +43,22 @@ func ConvertImageTagToImage(imageTag string) v1alpha1.Image {
 	return image
 }
 
+// ConvertImageToImageTag converts an Image into a plain string (domain/namespace/name:tag).
+func ConvertImageToImageTag(image v1alpha1.Image) string {
```

Comment:
Framework package should be agnostic of our domain. I will think on something one I get back.